static/usr/lib/systemd/system/snapd.autoimport-device@.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Auto import assertions from a specific block device
+After=snapd.service snapd.socket snapd.seeded.service
+# don't run on classic
+ConditionKernelCommandLine=|snap_core
```

Comment:
We don't have `snap_core` in the kernel command line in UC20+ iirc. I see:
`snapd_recovery_mode=run console=ttyS0 console=tty1 panic=-1`
I don't think we need to do this backward compatible.

Reply:
Right. I copied that from from `snapd.autoimport.service`. I will fix that.

Reply:
Removed